src/setup-java.ts

```diff
@@ -1,60 +1,69 @@
 import * as core from '@actions/core';
-import * as installer from './installer';
+import * as installer from './installer-updated';
 import * as auth from './auth';
 import * as gpg from './gpg';
-import * as constants from './constants';
+import {
```

Comment:
I think the previous approach with `import * as constants from './constants';` is more readable